src/RepoDriver.sol

```diff
@@ -78,13 +81,15 @@ contract RepoDriver is ERC677ReceiverInterface, ERC2771Context, Managed {
         mapping(bytes32 requestId => uint256 userId) requestedUpdates;
         /// @notice The new address of the AnyApi operator.
         OperatorInterface operator;
-        /// @notice The fee in Link for each user owner
+        /// @notice The fee in Link for each user owner.
```

Comment:
I think one of the things I admire in your work is your passion for detail.
(Assuming this is not some prettier library)